src/Compilers/CSharp/Portable/Binder/LocalScopeBinder.cs

```diff
@@ -204,8 +204,20 @@ protected ImmutableArray<LocalSymbol> BuildLocals(SyntaxList<StatementSyntax> st
                         {
                             Binder localDeclarationBinder = enclosingBinder.GetBinder(innerStatement) ?? enclosingBinder;
                             var decl = (LocalDeclarationStatementSyntax)innerStatement;
-                            LocalDeclarationKind kind = decl.IsConst ? LocalDeclarationKind.Constant : LocalDeclarationKind.RegularVariable;
-                            foreach (var vdecl in decl.Declaration.Variables)
+                            LocalDeclarationKind kind;
```

Comment:
I don't think these binder changes should be included. Right now we're only testing parsing.